docs/rackhd/install_os.rst

```diff
@@ -0,0 +1,116 @@
+dnsServers          Array            *optional*   If specified, this contains an array of objects, each object contains the Domain Name Servers, multiple DNS servers could be specified, the first one will be primary, others are alternative.
+networkDevices      Array            *optional*   The static IP setting for network devices after OS installation. If it is omitted, null or 0 length, RackHD will not touch any network devices setting, so all network devices remain at the default state (usually default is DHCP).If there is multiple setting for same device, RackHD will choose the last one as the final setting, both ipv4 and ipv6 are supported here.
+rootSshKey          String           *optional*   The SSH key that will be appended into the file /root/.ssh/authorized_keys in target OS.
+installDisk         String/Number    *optional*   The value could be string or integer number. For string, it could be the value like '/dev/sda' or the value of 'esxiWwid'(for ESXi), 'linuxWwid'(for Linux) , For number, it's the value of 'identifier' index. If *installDisk* not specified, OS will be installed into default SATADOM disk, and if there's no SATADOM disk, OS will be installed into 'identifier' 0 disk. **NOTE:** the value of 'esxiWwid', 'linuxWwid', 'identifier' could be got from node catalog's 'driveId' source.
+kvm                 Boolean          *optional*   The value is true/false, If *kvm* is omitted, null or 0 length, kvm will not be installed.  indicates to install kvm or not, default is false.
+switchDevices       Array            *optional*   **(ESXi only)** If specified, this contains an array of objects with switchName and uplinks (optional) parameters.  If *uplinks* is omitted, null or 0 length, the vswitch will be created with no uplinks.
+postInstallCommands Array            *optional*   **(ESXi only)** If specified, this contains an array of string commands that will be run at the end of the post installation step.  This can be used by the customer to tweak final system configuration
+
+=================== ================ ============ ============================================
+
+For **users** in payload:
+
+=========== ======== ============ ============================================
+Parameters  Type     Flags        Description
+=========== ======== ============ ============================================
+name        String   **required** The name of user
+password    String   **required** The password of user, it could be clear text, RackHD will do encryption before store it into kickstart file.
+uid         Number   **required** The unique identifier of user.
+sshKey      String   *optional*   The SSH key that will be appended into the file ~/.ssh/authorized_keys in target OS.
+=========== ======== ============ ============================================
+
+For **networkDevices** in payload, both ipv4 and ipv6 are supported:
+
```

Comment:
No mention of 'device' parameter?
Additional esxi only parameter:
esxSwitchName, string, optional, The vswitch to attach the vmk device to.  vSwitch0 is used by default if no esxSwitchName is specified